src/main/java/LacksDataBaseProject/AddAndRemoveUser.java

```diff
@@ -0,0 +1,48 @@
+package LacksDataBaseProject;
+
+import java.util.Scanner;
+
+public class AddAndRemoveUser {
+    static Scanner input = new Scanner(System.in);
+
+    public static void addUser(User user) {
+        System.out.println("Name: ");
+        String userName = input.next();
+        System.out.println("SkypeID: ");
+        String skypeId = input.next();
+        System.out.println("Login: ");
+        String login = input.next();
+        System.out.println("Password: ");
+        String password = input.next();
+        System.out.println("Repeat password: ");
+        String password2 = input.next();
+        if (password.equals(password2)) {
+            while (true) {
+                System.out.println("Select user roles: ");
+                for (int i = 0; i < 2; i++) {
+                    System.out.println("\t" + (i+1) + ". " + Role.values()[i]);
+                }
+                int addUserRole = input.nextInt();
+                Role role = Role.values()[addUserRole - 1];
+                User newUser = new User(userName, login, password, skypeId, role);
+                User.listOfUsers.add(newUser);
+                break;
+            }
+        }else {
+            System.out.println("Password is not correct.");
+        }
+    }
+
+    public static void removeUser () {
+            System.out.println("Give skype ID, to remove: ");
+            String skypieID = input.next();
+            for (User user : User.listOfUsers)
+                if (user.getSkypeID() == skypieID && user.getRole() != Role.ADMIN) {
```

Comment:
No wlasnie - tu chyba sprawdzasz role tego ktory jest do wywalenia z listy a to powienien byc sprawdzany ten który usuwa nie?